web/tests/message_list_view.test.js

```diff
@@ -764,7 +764,7 @@ test("render_windows", ({mock_template}) => {
     }
 
     function verify_no_move_range(start, end) {
-        // In our render window, there are up to 300 positions in
+        // In our render window, there are up to 250 positions in
```

Comment:
Read comments when you're updating them; This is not correct -- 250 - 50 x 2 = 150.

Reply:
I did read the comment. It seemed wrong (sorry for not explicitly commenting about that) because there were calls like `verify_no_move_range(0, 350);`. I was still wrong though, and I understand how it works better now than when I changed the comment. I can change it to something like "up to 150 positions, with potentially 50 on the lower end if it starts at 0"

Reply:
Yeah this seems fine.